pkg/ovs/ovsctl/ofctl.go

```diff
@@ -110,6 +110,11 @@ func (c *ovsCtlClient) DumpGroups(args ...string) ([][]string, error) {
 	return groupList, nil
 }
 
+func (c *ovsCtlClient) AddMeterEntry(meterId uint32, rate uint32) error {
+	_, err := c.RunOfctlCmd("add-meter", fmt.Sprintf("meter=%d,pktps,band=type=drop,rate=%d", meterId, rate))
```

Comment:
sorry I missed this earlier. We no longer use ovs-ofctl for flow programming AFAIK, why not add this support to ofnet / libOpenflow?

Reply:
That will be more complicated. Previously, I thought we simply use quite basic functions of OF meter, which are `add meter entry` and `apply meter entry in flows`. And for adding meter entry we only add them once while init and I saw there is an interface of run ofctl cmd. So I use the above way to impl adding process.
If it is the case as you said, I could add support in ofnet and libOpenFlow. What do you think?

Reply:
maybe @tnqn and @wenyingd can comment, but adding support to ofnet / libOpenflow seems like the right thing to do

Reply:
Got it. PR on libOpenflow opened. And changes needed on ofnet are also ready.